src/test/resources/application-integration-test.yml

```diff
@@ -40,3 +40,9 @@ api:
     sts.assume.role: arn:aws:iam::1234567890987:role/grant-me-access-to-ier
 
 caching.time-to-live: PT2S
+
+jobs:
+  enabled: false
+  pending-download-monitoring:
+    expected-maximum-pending-period: 24H
```

Comment:
Once the above typo is fixed, I think this change to the period is going to break the tests